src/Constants.ts

```diff
@@ -6,4 +6,10 @@ export const CATALOG_FILE = 'Catalog.json';
 export const CUSTOM_PATHS_FILE = 'CustomPaths.json';
 export const CONFIG_FILE = 'Config.json';
 
-export const TYPESCRIPT_RULE_PREFIX = '@typescript';
\ No newline at end of file
+export const TYPESCRIPT_RULE_PREFIX = '@typescript';
+
+export enum ENGINE {
```

Comment:
Standardizing engine names as enum so that the usage is always uniform.

Reply:
I don't object to this, but it seems slightly excessive.